canonical-web-and-design/microstack.run
# Flask conversion + Add discourse module

## Done

- Flask conversion
- Add discourse module

## QA

- Use the demo https://microstack-run-55.demos.haus/ and https://microstack-run-55.demos.haus/docs
OR
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8039/docs
- Some things worth checking:
 - [x] Global nav appears
 - [x] Javascript: copy to clipboard; toggle windows, linux instruction, os selector; docs navigation work properly
 - [x] Any console errors
 - [x] Visually it should look almost exactly the same as https://microstack.run/
 - [x] Navigate through `/docs` see if you encounter any issues

Optional: Review the site again with docker:
1. `DOCKER_BUILDKIT=1 docker build --tag microstack-run .`
2. `docker run -d -p 9000:80 --env SECRET_KEY=insecure_secret_key microstack-run`
3. Now check http://localhost:9000, check points before mentioned:

Discourse data is here https://discourse.ubuntu.com/c/microstack/docs


## Issue / Card

Fixes #51 #53
